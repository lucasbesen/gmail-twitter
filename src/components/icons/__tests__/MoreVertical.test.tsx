import React from 'react';
import renderer from 'react-test-renderer';

import { MoreVertical } from '../';

it('should render MoreVertical icon without error', () => {
  const component = renderer.create(<MoreVertical />);
  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
