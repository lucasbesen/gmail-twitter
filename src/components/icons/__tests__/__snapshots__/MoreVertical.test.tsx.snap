// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render MoreVertical icon without error 1`] = `
<svg
  aria-hidden="true"
  className="sc-EHOje eRdHEU StyledIconBase-sc-bdy9j4 jbPeIR"
  fill="currentColor"
  focusable="false"
  viewBox="0 0 24 24"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M12 8c1.1 0 2-.9 2-2s-.9-2-2-2-2 .9-2 2 .9 2 2 2zm0 2c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm0 6c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2z"
  />
</svg>
`;
